packages/aws-cdk-lib/aws-events-targets/README.md

```diff
@@ -389,6 +390,29 @@ rule.addTarget(
 );
 ```
 
+### Launch type for ECS Task
```

Comment:
In the next section "Assign public IP addresses to tasks", the word launch type appears, so I put it here.